content/documentation/pages/2-concepts/1-architecture.md

```diff
@@ -1,33 +1,330 @@
 ---
 path: 'concepts/architecture/'
 title: 'Architecture'
-description: 'Lorem markdownum madefacta, circumtulit aliis, restabat'
+description: "Introduction to Data Flow's Architecture."
 ---
 
 # Architecture
 
-TODO
+Thi guide explains the main concepts of Data Flow's architecture:
 
-## System Requirements
+- Data Flow's Server Components
+- The types of applications the server components can deploy for Streams and Batch Jobs
+- The microservice architecture of the deployed applications and how that is defined using a DSL
+- The platforms where they are deployed
 
-**Java:** Data Flow uses Java 8.
+Other sections of the guide explain
 
-**Database:** The Data Flow Server and Skipper Server need to have an RDBMS installed.
+- How the Server Components are secured and the tooling used to interact with them.
+- Runtime monitoring of Stream data pipelines.
+- The Spring projects that you can use to develop Stream and Batch data pipelines
+
+## Server Components
+
+Data Flow has two main components:
+
+- Data Flow Server
+- Skipper Server
+
+The main entry point to access Data Flow is through the Web API of the Data Flow Server.
```

Comment:
We can possibly say that Spring Cloud Data Flow server exposes RESTful API and we have REST client interface to access the REST API. The Web Dashboard and the Shell are some of the implementations of the REST client interface.

Reply:
i'll add RESTful back in, already mention the shell and web dashboard accessing data flow server via the rest api in the next sentence.